### ISSUE TITLE
Correct indentation in blog post template

### ISSUE THREAD

**Issue by drtutut** [drtutut/ewo#5]
Indentation is not correct, eg: 
```
#+TITLE:        Un saxophone en bambou
#+SUBTITLE:    Les perces côniques font rien qu'à m'embêter
#+AUTHOR:      EricW
#+EMAIL:       ----@----.--
#+DATE:        <2018-05-30 mer.>
#+KEYWORDS:    saxophone bambou, bambou, anche simple, lutherie sauvage
#+DESCRIPTION: Les perces côniques font rien qu'à m'embêter
#+LANGUAGE:    en
#+OPTIONS: H:3 num:nil toc:nil \n:nil ::t |:t ^:nil -:nil f:t *:t <:t
#+FILETAGS: :saxophone_bambou:bambou:anche_simple:lutherie_sauvage:
#+EWO_STATE: published
```

**Comment by drtutut** [comment]
Corrected. Will be available in 3.4.1.

**Comment by drtutut** [comment]
Corrected. Will be available in 3.4.1.
